runtime/j9vm/java11vmi.c

```diff
@@ -417,7 +417,13 @@ addModuleDefinition(J9VMThread * currentThread, J9Module * fromModule, const cha
 	J9ClassLoader * const classLoader = fromModule->classLoader;
 
 	UDATA retval = ERRCODE_GENERAL_FAILURE;
-	if (!areNoPackagesDefined(currentThread, classLoader, packages, numPackages)) {
+
+	/* the bootstrap classloader is exempt from unnamed module checks
+	 * because it may place classes in the unnamed
+	 * module before java.base is created.
+	 */
```

Comment:
I'm still concerned about this logic.  Is there a more fundamental issue in how we're "fixing up" the classes loaded prior to java.base being available?

Reply:
The comment was misleading; I have reworded it.